html/comparison.html.haml

```diff
@@ -29,7 +43,7 @@
 
     %script#scope_os.partial{:type => "text/ng-template"}
       %a.scope_anchor{:id => "os"}
-      #os_container.scope
+      #os_container.scope{"ng-show" => "diff.os"}
```

Comment:
Do we still need those show conditions now that the scopes are only included if they exist in the diff anyway?